execution/executor/src/lib.rs

```diff
@@ -65,30 +65,30 @@ where
             .expect("Failed to read startup info from storage.");
 
         let (
+            version,
             state_root_hash,
             frozen_subtrees_in_accumulator,
-            num_elements_in_accumulator,
             committed_timestamp_usecs,
             committed_block_id,
         ) = match startup_info {
             Some(info) => {
                 info!("Startup info read from DB: {:?}.", info);
                 let ledger_info = info.ledger_info;
                 (
+                    Some(ledger_info.version()),
```

Comment:
Isn't this at odds with the fix in https://github.com/libra/libra/pull/511 ?

Reply:
Yes, I made a mistake.